packages/drivers/odsp-driver/src/index.ts

```diff
@@ -31,7 +31,3 @@ export * from "./odspDriverUrlResolver";
 
 // It's used by URL resolve code, but also has some public functions
 export * from "./odspFluidFileLink";
-
-// Wire format related
-export * from "./compactSnapshotParser";
```

Comment:
Push uses the code. We need to export it.

Reply:
@GaryWilber, can you please glance?
It's not really great that we export such low-level things, but I guess it's required.
I'll add export, but it's shaped changed. Technically it's breaking change and needs to move to next branch, but that will add substantial amount of time till we can release it, and I'd love to get data sooner and enable it in prod.

Shape changed from 

> function parseCompactSnapshotResponse(buffer: ReadBuffer): ISnapshotContents;

to 

>function parseCompactSnapshotResponse(buffer: Uint8Array, logger: ITelemetryLogger): ISnapshotContents;

Note - there were couple other perf changes merged in already, main one showing big improvements - https://github.com/microsoft/FluidFramework/pull/12054

Reply:
Yea Push needs this export. I can handle the updates my side when pulling in a new version of the packages. It shouldn't be necessary to mark this as a breaking change since it should only affect Push.